demo/server.js

```diff
@@ -1,10 +1,10 @@
 const express = require('express');
 const fs = require('fs');
+const { waitForDebugger } = require('inspector');
```

Comment:
Delete this line. It's fine if you do this locally but you should not commit debugger stuff.